src/app/beer_garden/api/stomp/transport.py

```diff
@@ -207,7 +209,7 @@ def connect(self) -> bool:
 
                 self.conn.subscribe(
                     destination=self.subscribe_destination,
-                    id=self.username,
+                    id="".join([choice(ascii_letters) for _ in range(10)]),
```

Comment:
These changes are making the assumption that the topic being connected to is not durable. If the topic is durable, every connection will appear as a new user. So any messages queued up for the previous ID will just sit there. Are we making that decision that this should be our standard operating procedure?